build.zig

```diff
@@ -6,6 +6,14 @@ pub fn build(b: *std.Build) void {
     const optimize = b.standardOptimizeOption(.{});
     const target = b.standardTargetOptions(.{});
 
+    const lib = b.addStaticLibrary(.{
+        .name = "clap",
+        .root_source_file = .{ .path = "clap.zig" },
+        .target = target,
+        .optimize = optimize,
+    });
+    b.installArtifact(lib);
```

Comment:
This will build a static library, which makes no sense for zig-clap. We export no functions that can be found and linked to. The `addModule` at the top of `build` should be enough for the package manager for pure zig packages.

Reply:
@Hejsil You're right. I was confused. Will remove them. :)

Reply:
@Hejsil You're right. I was confused. Removed them. :)